frontend/src/app/spot/styles/sass/variables/zindex.sass

```diff
@@ -1,6 +1,6 @@
 @use "sass:map"
 
-$spot-z-indexes: ("toast": 300, "tooltip": 500, "header": 800, "drop-modal": 1000)
+$spot-z-indexes: ("toast": 10, "tooltip": 500, "header": 800, "drop-modal": 1000)
```

Comment:
Please move the z-index for the side menu here too so that we have an overview of what goes under and over what. reducing the toast z-index to 10 feels wrong. I'd rather keep it at 300 and make sure the sidemenu is at 400 or 700, depending on whether it should be above or below tooltips. 

Whether the sidemenu should be above a tooltip or below it is a question I'd defer to @marcalcobe @psatyal

Reply:
@b12f Thanks for the insight, I went with adding 400 to the main menu, keeping it under the tooltip, however I will adjust it once I have a feedback from the designers.

Reply:
Hi @b12f and @dombesz. OK for letting the toast go _under_ the sidebar menu. I agree that despite the toast being dismissible, if someone opens the menu, that user action should take precedence.